packages/vehicles/src/utils.js

```diff
@@ -0,0 +1,299 @@
+/* eslint-disable no-unused-vars */
+import polyline from "@mapbox/polyline";
+
+/** deep copy */
+export function deep(obj) {
+  return JSON.parse(JSON.stringify(obj));
+}
+
+/**
+ * will take an input object (e.g., probably a defaultProp representing a leaflet style),
+ * deep copy that object, and return back a new obj with the .color set
+ *
+ * @param color
+ * @return deep copied object with color set
+ */
+export function setColor(color, obj) {
+  const retVal = deep(obj);
+  retVal.color = color;
+  return retVal;
+}
+
+/** return map zoom value from leaflet */
+export function getZoom(leaflet, defZoom = 15) {
+  let retVal = defZoom;
+  try {
+    retVal = leaflet.map.getZoom();
+  } catch (e) {
+    console.log(e);
+  }
+  return retVal;
+}
+
+/** check 360 heading for a valid number */
+export function checkHeading(heading) {
```

Comment:
What values have you seen for heading?
(Depending on the values it might make more sense to do a modulo 360.)

Reply:
I think I'm being overly cautious, since the markers rotate and I didn't want there to be any funky data to muck with them. I'd rather leave as it, since I can't quite remember why it's there and the way it is.

Reply:
I'm good with that.